customer-commons/build.gradle

```diff
@@ -16,9 +16,16 @@ dependencies {
 
 
 
-
+    testImplementation project (":customer-testing")
     testImplementation (group: 'com.amazonaws', name: 'DynamoDBLocal', version: project.ext.DynamoDBLocalVersion) {
         exclude group: 'com.amazonaws', module: 'aws-java-sdk-dynamodb'
         exclude group: 'com.amazonaws', module: 'aws-java-sdk-core'
     }
+}
+
+
+test{
+    environment "TABLE_NAME","nva_customers"
```

Comment:
there is no reason for the indices to be set by ENV variables. The nature of the indices (and therefore their name) depends on their use by the code. Changing their name has no use. In next PR I will clean this up.